### PR TITLE
test(next-version): add test for scope with whitespace

### PR DESCRIPTION
**Pull request by orhun**
See #1996

### CODE REVIEW

**Comment by lgatellier** [comment]
@orhun I think you can also add a test with minor a non-breaking change, since the bug also occurs in this case.

File: crates/next_version/tests/all/normal.rs [hunk 1]
```diff
@@ -94,3 +94,23 @@ fn commit_with_custom_minor_increment_regex_increments_minor_version() {
         Version::new(1, 3, 0)
     );
 }
+#[test]
```
Comment (marcoieni) [inline crates/next_version/tests/all/normal.rs:97]:
```suggestion

#[test]
```

**Review by marcoieni — APPROVED** [review]
